components/FooterSection.js

```
@@ -0,0 +1,139 @@
+              </a>
+              <a
+                href="https://www.instagram.com/erp.solver"
+                target="_blank"
+                className="w-10 h-10 border-2 border-white rounded-full flex items-center justify-center hover:bg-white hover:text-teal-500 transition-all duration-300 group"
+                aria-label="Instagram"
+              >
+                <FaInstagram className="w-5 h-5" />
+              </a>
+              <a
+                href="https://www.linkedin.com/in/speed-solver-56578a99/"
+                target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)